src/Symfony/Component/AssetMapper/Command/AssetMapperCompileCommand.php

```diff
@@ -89,7 +89,7 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         if ($this->isDebug) {
             $io->warning(sprintf(
-                'You are compiling assets in development. Symfony will not serve any changed assets until you delete the files in the "%s" directory.',
+                'You are compiling assets in an environment with debug mode enabled. Symfony will not serve any changed assets until you delete the files in the "%s" directory again.',
```

Comment:
```suggestion
                'You are compiling assets with debug mode enabled. Symfony will not serve any changed assets until you delete the files in the "%s" directory again.',
```

Reply:
This sounds like debug mode is a parameter of `asset-map:compile`. I changed it again - what do you think?